operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -373,14 +373,16 @@ metadata:
     categories: Streaming & Messaging
     certified: 'false'
     containerImage: registry.redhat.io/amq7/amq-streams-rhel8-operator@sha256:c1a98649ba33febede789ac26d541632ff979cc8817bdd9f6188ffa755e6c7ec
-    createdAt: 2022-02-12 23:02:00
+    createdAt: 2022-07-01 10:33:00
     description: Red Hat AMQ Streams is a massively scalable, distributed, and high performance data streaming platform based on Apache Kafka®.
     repository: https://github.com/strimzi/strimzi-kafka-operator
     support: Red Hat
     operators.openshift.io/infrastructure-features: |-
       ["Disconnected", "Proxy"]
     operators.openshift.io/valid-subscription: |-
       ["Red Hat Integration", "Red Hat AMQ"]
```

Comment:
I guess we should add `Red Hat Application Foundations` here?

Reply:
I was not aware of it, going to add.